### ISSUE TITLE
Test card icons should have valid alt text & not hidden

### ISSUE THREAD

**Issue by nathancrtr** [CDCgov/prime-simplereport#4051]
## Area or Component
Conduct test

## Quick Summary
* The following test card icons do not have alternative text:
    * Close icon
    * Device info circle icon
    * Timer icon

* Additionally, the SVG elements for these icons are marked as `aria-hidden=true`, making them inaccessible to assistive technology
    * The icon _itself_ is focusable, so this may not be an issue

## Accessibility Violation
* `alt-text-missing`
* The text alternative is missing.

## Action Requested
Test card icon HTML elements should each have valid `role` and `aria-label` attributes.

Additionally, the SVG for the icons should not be hidden and should be focusable as well.

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/f11273c49f8e25f8a85449b510e6947a49d3457c/frontend/src/app/testQueue/QueueItem.tsx#L642-L656

https://github.com/CDCgov/prime-simplereport/blob/1cd52304de1569a1b66409777965d62ee8c66246/frontend/src/app/testQueue/TestTimer.tsx#L210

https://github.com/CDCgov/prime-simplereport/blob/a197d10d22d75ae212804b517ce11971ab4a7093/frontend/src/app/commonComponents/TextWithTooltip.tsx#L49

## Original Deque Issue
https://cdc-devtools.dequecloud.com/rule-help/en/alt-text-missing
